frontend/src/components/scratchpad/scratchpad.tsx

```diff
@@ -187,17 +187,19 @@ export const ScratchPad: React.FC = () => {
               className="border rounded-md hover:shadow-sm cursor-pointer hover:border-input overflow-hidden"
               onClick={() => handleSelectHistoryItem(item)}
             >
-              <LazyAnyLanguageCodeMirror
-                language="python"
-                theme={theme}
-                basicSetup={{
-                  highlightActiveLine: false,
-                  highlightActiveLineGutter: false,
-                }}
-                value={item.trim()}
-                editable={false}
-                readOnly={true}
-              />
+              <Suspense>
+                <LazyAnyLanguageCodeMirror
```

Comment:
Shouldn't suspense be on the component level instead?

Reply:
it needs to go above the component in the tree that is being lazily-loaded